packages/esm-appointments-app/src/patient-queue/visit-form/queue.resource.ts

```diff
@@ -36,7 +36,7 @@ export async function saveQueueEntry(
         patient: {
           uuid: patientUuid,
         },
-        startedAt: toDateObjectStrict(toOmrsIsoString(new Date())),
+        startedAt: new Date(),
```

Comment:
Most of the changes in this PR are this.  Find/replace wherever we are converting from date->string->date and just using the original date instead.